libs/designer/src/lib/ui/panel/templatePanel/quickViewPanel/tabs/summaryTab.tsx

```diff
@@ -144,7 +144,7 @@ export const summaryTab = (
     }),
     secondaryButtonOnClick: () => {
       dispatch(closePanel());
-      if (showCreate) {
+      if (clearDetailsOnClose) {
```

Comment:
I see what this is doing here; only for the multi-workflow scenario, we clear the details. 

Before, for single workflow, we wiped the details in the state; so when we run "changeCurrentTemplateName" then "loadTemplate", it won't cause flickering since it goes from _no data_ to _loaded data_.

Since here, the state for the previously selected tab is preserved, I am worried that the flickering may happen for loading a new single template to display info for.

Reply:
This is only going to clear the details when caller passes it, for multi workflow we are not clearing the details because it is being opened from templates overview page when we click on single workflow, for everything else the caller should be passing in true